auth0/src/main/java/com/auth0/android/authentication/AuthenticationException.java

```diff
@@ -157,17 +157,23 @@ public boolean isInvalidConfiguration() {
 
     /// When MFA code is required to authenticate
     public boolean isMultifactorRequired() {
-        return "a0.mfa_required".equals(code);
+        return "mfa_required".equals(code) || "a0.mfa_required".equals(code);
     }
 
     /// When MFA is required and the user is not enrolled
     public boolean isMultifactorEnrollRequired() {
-        return "a0.mfa_registration_required".equals(code);
+        return "a0.mfa_registration_required".equals(code) || "unsupported_challenge_type".equals(code);
```

Comment:
The error is the specific challenge type is not expected vs enrolment in general. Should we clarify this? As it could be they don't support OTP but do OOB?

Reply:
`association_required` will be the new not-enrolled error code once they release the new API, currently this is `unsupported_challenge_type`. On the other side, "unsupported_challenge_type" _does_ mean that the challenge is not supported. Because we're not going to support other challenge type than OTP, I don't think that's an error we can provide an alternate path to the user to continue the authentication. I tried to find without success, but I think somewhere in the docs they mention that OTP must be supported at least.